### ISSUE TITLE
Doesn't seem to build using the latest version of the sdfat arudino library

### ISSUE THREAD

**Issue by Monsjakon** [djtersteegc/smartportsd#4]

**Comment by hexbus** [comment]
@monsjakon,

Yep, you need to backlevel it per here:  https://tinkerdifferent.com/threads/smartportsd-lets-build-it.454/post-34114

@djtersteegc - 

Adding on to this, I've built several of these, and made a custom 9 pin to DB19 adapter per your pinout.  They can compile when I backlevel sdfat per that thread above with your codebase (or if I use the @erichelgeson build [here ](https://github.com/erichelgeson/smartportsd).  I can upload just fine, and I can put images on the device just fine.  However, when I boot my II GS, which is set to boot from device 5 (Smartport), it can't find a bootable device.  Even PR#5 won't work.

When I hook up my FujiNet Apple (which I also believe is your hardware) IDC20 model - (which I borrowed your DB19 to IDC20 board files to make my 9 pin adapter), from [here](https://github.com/FujiNetWIFI/fujinet-hardware/tree/master/AppleII/FujiApple-DevKit-IDC-20-Rev1), it boots just fine from Device 5 (Smartport).  Nothing else connected just like I was trying the SmartPort SD.

I did notice that Pin 15 (WREQ) and 17 (DRIVE1) aren't connected to anything on the 9 pin header.  Does this impact anything on why the system can't find it?  I've built two, and our local computer group has built them as well, and is running into the same issue, and we're trying to figure out what to do next to help troubleshoot to help others like [here ](https://tinkerdifferent.com/threads/smartportsd-lets-build-it.454/post-15483)who are having the same problem.

Thanks for all your work with these projects!

**Comment by colinleroy** [comment]
> Adding on to this, I've built several of these, and made a custom 9 pin to DB19 adapter per your pinout. They can compile when I backlevel sdfat per that thread above with your codebase (or if I use the [@erichelgeson](https://github.com/erichelgeson) build [here ](https://github.com/erichelgeson/smartportsd). I can upload just fine, and I can put images on the device just fine. However, when I boot my II GS, which is set to boot from device 5 (Smartport), it can't find a bootable device. Even PR#5 won't work.

I suspect it is because the SD card init code takes too long to handle the early Smartport RESET/ENABLE signals.
What happens if you let it fail to boot, then reset using Ctrl-Open-Apple-Reset ?
